app/assess/models/fund.py

```diff
@@ -20,6 +21,10 @@ def from_json(data: dict):
             id=data.get("id"),
             description=data.get("description"),
             short_name=data.get("short_name"),
+            all_uploaded_documents_section_available=data.get(
+                "all_uploaded_documents_section_available"
+            )
+            or False,
```

Comment:
Do we need this False here?

Reply:
Hmm - just a default in-case anyone calling the method on their own branches and don't have the changes!